covid-frontend/src/components/common/Tag/styles.module.scss

```diff
@@ -8,19 +8,19 @@
   text-align: center;
   white-space: nowrap;
 
-  &.red {
-    background-color: $error-light2;
-    color: $error;
-    @include border($error-light, 4px);
-  }
-  &.orange {
-    background-color: $warning-light2;
-    color: $warning;
-    @include border($warning-light, 4px);
-  }
-  &.green {
-    background-color: $success-light2;
-    color: $success;
-    @include border($success-light, 4px);
-  }
+  // &.red {
```

Comment:
Comments should always have a "documented reason to stay" so the next engineer would know if they can be remvoed or not.